modules/coreos-layering-configuring-on.adoc

```diff
@@ -23,16 +37,47 @@ You need a separate `MachineOSConfig` CR for each machine config pool where you
 
 :FeatureName: On-cluster image layering
 include::snippets/technology-preview.adoc[]
+include::snippets//coreos-layering-configuring-on-pause.adoc[]
+
+In the case of a build failure, for example due to network issues or an invalid secret, the MCO retries the build three additional times before the job fails. The MCO creates a different build pod for each build attempt. You can use the build pod logs to troubleshoot any build failures. However, the MCO automatically removes these build pods after a short period of time. 
+
+.Example failed `MachineOSBuild` object
+[source,terminal]
+----
+NAME                                       PREPARED   BUILDING   SUCCEEDED   INTERRUPTED   FAILED
+layered-c8765e26ebc87e1e17a7d6e0a78e8bae   False      False      False        False        True
+----
+
+// Not in 4.18; maybe 4.19
+// You can manually rebuild your custom layered image by either modifying your `MachineOSConfig` object or applying an annotation to the `MachineOSConfig` object as discussed in the following section.
+
+[discrete]
+[id="coreos-layering-configuring-on-limitations_{context}"]
+== On-cluster layering Technology Preview known limitations
```

Comment:
This is a procedure module, so I don't think we're supposed to be adding subheadings like this in the same file. This should be moved to its own concept module.

Edit: I actually just noticed that this is an enormous procedure module. That it continues down after all this.  So it seems like it definitely should be broken out/broken up better